### ISSUE TITLE
[Query] Any new CNI release planned in coming days?

### ISSUE THREAD

**Issue by sushanthakumar** [containernetworking/cni#610]
Just curious to know if any new CNI release coming up anytime in near future
Thank u guys

**Comment by bboreham** [comment]
Yes, we need to release a new version of the spec with the GET function.

We figured we would get it working in the base plugins (https://github.com/containernetworking/plugins) before setting in stone.

So "near future" but probably more like a few weeks than a few days.
